content/working_with/console/customization/edit-mode.md

```diff
@@ -0,0 +1,89 @@
+
+**Edit Mode** enables you to create new dashboard pages, add or remove widgets and manage how widgets are displayed on a page.
+
+{{% note %}}
+If you have a `user` role, your ability to create dashboard pages and manage widgets depends on the configuration permissions that have been set by the administrator.
+{{% /note %}} 
+
+
+## Entering Edit Mode
+
+Click the dropdown arrow next to your user name and select **Edit Mode**. 
+
+![Edit Mode - menu]( /images/ui/customization/edit-mode_menu.png )
+
+{{% note %}}
+If you do not see **Edit Mode** in the dropdown menu, you do not have permissions to edit configuration or add pages.
+{{% /note %}} 
+
+When you have entered Edit mode, the **Add Widget**, **Add Tabs**, **Add Page**  and **Exit** buttons appears on the screen. In addition, you can now move widgets already on the page by clicking on their title bar and dragging them to the preferred position. You can also remove them from the page by clicking the **X** in the right corner of the widget (visible when hovering over the widget's title bar), or click the widget configuration button.
+
+![Edit Mode - menu]( /images/ui/customization/edit-mode_overview.png )
+
+
+## Configuring Widgets
+
+Some widgets have configuration option that you can define or edit.
+
+![Configure Widget dialog]( /images/ui/customization/configure-widget.png )
+
+ 1. In **Edit Mode**, hover over the title bar of the widget you want to configure and click the gear icon ![Gear Icon]( /images/ui/icons/gear-icon.png ).
+ 2. Make your required changes and click **Save**.   
+    
+The available fields are widget-specific.
+
+{{% tip title="Tip" %}}
+You can choose to display Blueprints Catalog widget contents as a table.
+{{% /tip %}}
+
+
+## Adding Widgets
+
+A catalog of widgets is available to enable you to select your preferred data display on any page.
+
+1. In **Edit Mode**, click **Add Widget**.   
+   The **Add Widget** button is visible on all pages when you are in **Edit Mode**.  
+2. Select the widget that you want to add and click **Add selected widgets**.   
+   You can enter free text in the search box to find a widget.
+
+![Edit Mode - Add widget modal]( /images/ui/customization/edit-mode_add-widget-modal.png )
+
+{{% tip title="Tip" %}}
+Click on widget thumbnail to see it full-sized.
+{{% /tip %}}
+
+The widget appears on the page. You can drag and drop it in the position that you prefer. You can resize a widget while holding the resize icon in the lower right corner of the widget.
+
+To delete a widget, click the **X** icon in the top right corner of the widget.
+
+
+## Adding Tabs
+
+1. In **Edit Mode**, click **Add Tabs**.  
+2. By default 2 new tabs will be added to the page. 
+
+![Edit Mode - Add widget modal]( /images/ui/customization/edit-mode_tabs.png )
+
+You can add new tab with **+** icon and remove tab using **X** icon nearby unwanted tab.
```

Comment:
You can add a new tab with the **+** icon and remove a tab using the **X** icon nearby the unwanted tab.